pyacq/dsp/tests/test_triggeraccumulator.py

```diff
@@ -0,0 +1,91 @@
+import time
+import numpy as np
+import pyqtgraph as pg
+
+from pyacq import create_manager, InputStream, NumpyDeviceBuffer, ThreadPollOuput
+from pyacq.dsp.trigger import AnalogTrigger, DigitalTrigger
+from pyacq.dsp.triggeraccumulator import TriggerAccumulator
+
+
+from pyqtgraph.Qt import QtCore, QtGui
+
+
+nb_channel = 2
+sample_rate =1000.
+chunksize = 100
+
+length = int(sample_rate*20)
+t = np.arange(length)/sample_rate
+buffer = np.random.rand(nb_channel, length)*.3
```

Comment:
I am stupid. This random do impact the test. because the thresh hold is 1.
I ptach this.